superset/assets/visualizations/deckgl/geojson.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { GeoJsonLayer } from 'deck.gl';
+
+import DeckGLContainer from './DeckGLContainer';
+
+
+function DeckGeoJsonLayer(slice, payload, setControlValue) {
+  const fd = slice.formData;
+  const fc = fd.fill_color_picker;
+  const sc = fd.stroke_color_picker;
+  const data = payload.data.geojson.features.map(d => ({
+    ...d,
+    properties: {
```

Comment:
this will fully override properties ind `d` which is not what we want. We should enforce the precedence rule here.